src/hotspot/share/gc/shenandoah/shenandoah_globals.hpp

```diff
@@ -451,6 +451,12 @@
   product(bool, ShenandoahAllocFailureALot, false, DIAGNOSTIC,              \
           "Testing: make lots of artificial allocation failures.")          \
                                                                             \
+  product(uintx, ShenandoahCoalesceChance, 0, DIAGNOSTIC,                   \
```

Comment:
Should this read: "Abandon remaining mixed collections with this likelihood.  Following each mixed collection, we abandon all remaining mixed collection candidate regions with likelihood ShenandoahCoalesceChance." ?  (I don't think this is precisely abandon this percentage of mixed collections".)

Reply:
Yes, that's a more accurate description. I'll update this description.